aptos-move/framework/aptos-stdlib/sources/data_structures/smart_vector.move

```diff
@@ -74,6 +74,16 @@ module aptos_std::smart_vector {
         option::destroy_none(big_vec);
     }
 
+    /// Destroy a table completely when T has `drop`.
+    public fun destroy<T: drop>(v: SmartVector<T>) {
```

Comment:
Is there a way to also add a clear() that wipes all data with a mutable reference? cc @banool